ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml

```diff
@@ -42,8 +42,8 @@ SignTransactionModalBase {
     required property string cryptoFees
     required property double slippage
 
-    property string serviceProviderName: "Paraswap"
-    property string serviceProviderURL: "https://www.paraswap.io/" // TODO https://github.com/status-im/status-desktop/issues/15329
+    property string serviceProviderName
```

Comment:
We might as well make them `required` now, wdyt?

Reply:
make sense!

Reply:
done!